### DISCUSSION TITLE
Delay EIP component

### DISCUSSION THREAD

**Discussion by denibrek** [apache/camel-karavan#777] [Q&A] [answered]
Occasioanally I have found that simple "Delay" component is not available. There is only delay-action kamelet, but looks like it doesn't provide async processing (at least there is no option to choose between sync and async).

What is proposed way to add delays with Camel Karavan?

**Comment by davsclaus** [comment]
There is a delay EIP but for some weird reason its now show in Karavan @mgubaidullin  ???
https://github.com/apache/camel/blob/main/core/camel-core-model/src/generated/resources/org/apache/camel/model/delay.json

**Comment by mgubaidullin** [comment]
https://github.com/apache/camel-karavan/issues/806

**Comment by mgubaidullin** [comment]
New components require a bit more manual attention (ex. find and set an Icon) before they are available in Karavan

**Comment by mgubaidullin** [comment, marked as answer]
Fixed in the next version
